exercises/practice/space-age/.meta/example.c

```diff
@@ -1,21 +1,41 @@
 #include "space_age.h"
+#include <stdbool.h>
 
 /* Number of seconds in a Earth year */
-#define EARTH 31557600
+#define EARTH_YR_IN_SECONDS 31557600
```

Comment:
I hate to be picky, but I think it makes more sense to avoid the abbreviation here:

```suggestion
#define EARTH_YEAR_IN_SECONDS 31557600
```